packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -675,16 +672,19 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
   }
 
   /** Build up the pages that should be rendered. */
-  private _buildPages(items: any[], startIndex: number, renderCount: number): IListState {
+  private _buildPages(props?: IListProps): IListState {
+    let { items, startIndex, renderCount } = (props || this.props);
```

Comment:
I think we should be consistent in this method. Either use `(props || this.props)` for everything or keep the logic outside? Now we're getting some props from the `props` parameter and some from `this.props`

Reply:
Are you saying:

```tsx
props = props || this.props;
let { ... } = props;
```

Reply:
Either that, or the previous logic where we explicitly pass in the 3 values that could come from `props`/`this.props`

Reply:
addressed.